pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/export/tabular/TsvParticipantExporter.java

```diff
@@ -0,0 +1,62 @@
+package org.broadinstitute.dsm.model.elastic.export.tabular;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.List;
+import java.util.Map;
+
+import com.google.common.net.MediaType;
+import org.apache.commons.lang3.StringUtils;
+import spark.Response;
+
+public class TsvParticipantExporter extends TabularParticipantExporter {
+    public static final String DELIMITER = "\t";
+
+    public TsvParticipantExporter(List<ModuleExportConfig> moduleConfigs,
+                                  List<Map<String, String>> participantValueMaps, String fileFormat) {
+        super(moduleConfigs, participantValueMaps, fileFormat);
+    }
+
+
+    public void export(Response response) throws IOException {
+        setResponseHeaders(response);
+
+        PrintWriter writer = response.raw().getWriter();
+        List<String> headerRowValues = getHeaderRow();
+        List<String> subHeaderRowValues = getSubHeaderRow();
+
+        writer.println(getRowString(headerRowValues));
+        writer.println(getRowString(subHeaderRowValues));
+        for (Map<String, String> valueMap : participantValueMaps) {
+            List<String> rowValues = getRowValues(valueMap, headerRowValues);
+            String rowString = getRowString(rowValues);
+            writer.println(rowString);
+        }
+        writer.flush();
+    }
+
+    public void setResponseHeaders(Response response) {
+        response.type(MediaType.TSV_UTF_8.toString());
+        response.header("Access-Control-Expose-Headers", "Content-Disposition");
+        response.header("Content-Disposition", "attachment;filename=" + getExportFilename(fileFormat));
```

Comment:
I would apply template method design pattern here to eliminate obvious duplication, since last two lines are exactly the same as for `ExcelParticipantExporter`.

Reply:
I think keeping it very explicit and simple here is worth the cost of duplication.  If we add a third or fourth file type, then definitely let's template what we can